Sources/Widgets/Widgets3D/ShapeWidget/behavior.js

```diff
@@ -121,6 +113,117 @@ export default function widgetBehavior(publicAPI, model) {
     vec3.normalize(model.zAxis, zAxis);
   };
 
+  publicAPI.setLabelTextCallback = (callback) => {
```

Comment:
use 
```
macro.setGet(publicAPI, model, ['labelTextCallback', 'resetAfterPointPlacement',  'point1'...]);
```
?

Reply:
`macro.setGet` creates the methods on the widget itself, not on the widget's behavior.